nannyml_cloud_sdk/client.py

```diff
@@ -32,7 +32,7 @@ def get_client() -> Client:
     if nannyml_cloud_sdk.api_token:
         headers['Authorization'] = f"ApiToken {nannyml_cloud_sdk.api_token}"
 
-    transport = RequestsHTTPTransport(url=f"{nannyml_cloud_sdk.url}/api/graphql", headers=headers)
+    transport = RequestsHTTPTransport(url=f"{nannyml_cloud_sdk.url}/graphql", headers=headers)
```

Comment:
Was this change intended? Looks like something from testing

Reply:
Oh man, very good catch. I recall saying to myself "I won't forget about this right". 

I did.